web-client/integration-tests/journey/petitionerSearchesForUnassociatedSealedCase.js

```diff
@@ -0,0 +1,31 @@
+import { caseDetailHelper as caseDetailHelperComputed } from '../../src/presenter/computeds/caseDetailHelper';
+import { runCompute } from 'cerebral/test';
+import { withAppContextDecorator } from '../../src/withAppContext';
+
+const caseDetailHelper = withAppContextDecorator(caseDetailHelperComputed);
+
+export const petitionerSearchesForUnassociatedSealedCase = cerebralTest => {
+  return it('petitioner searches for unassociated sealed case', async () => {
+    cerebralTest.setState('caseDetail', {});
+    const sealedSeedCaseDocketNumber = '105-20';
+
+    await cerebralTest.runSequence('gotoDashboardSequence');
+
+    cerebralTest.setState('header.searchTerm', sealedSeedCaseDocketNumber);
```

Comment:
Instead of directly setting state, could this value be updated using a sequence? I think in general we want to try to avoid directly manipulating state in the tests :)